packages/api/src/struct/Auth.ts

```diff
@@ -184,7 +184,9 @@ export class Auth {
 			this.oauthRest.setToken(discordAccessToken);
 
 			const fetched = (await this.oauthRest.get(Routes.user())) as APIUser;
-			const rawGuilds = (await this.oauthRest.get(Routes.userGuilds())) as RESTGetAPICurrentUserGuildsResult;
+			const rawGuilds = (await this.oauthRest.get(
+				`${Routes.userGuilds()}?with_counts=true`,
```

Comment:
```suggestion
				Routes.userGuilds(), { query: makeURLSearchParams({ with_counts: true }) }
```

https://discordjs.dev/docs/packages/rest/main/makeURLSearchParams:Function